### PR TITLE
net-mail/swaks: New upstream version 20201010.0

### PR DESCRIPTION
**Pull request by alarig**
Package-Manager: Portage-3.0.4, Repoman-3.0.1
Signed-off-by: Alarig Le Lay <alarig@swordarmor.fr>

### CODE REVIEW

File: net-mail/swaks/swaks-20201010.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+inherit perl-functions
+
+DESCRIPTION="Swiss Army Knife SMTP; Command line SMTP testing, including TLS and AUTH"
+HOMEPAGE="http://www.jetmore.org/john/code/swaks"
+SRC_URI="http://www.jetmore.org/john/code/swaks/${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+IUSE="ssl"
```
Comment (juippis) [inline net-mail/swaks/swaks-20201010.0.ebuild:15]: This doesn't seem to have any function in this ebuild, move it to `optfeature` instead.
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

File: net-mail/swaks/swaks-20201010.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit perl-functions
+
+DESCRIPTION="Swiss Army Knife SMTP; Command line SMTP testing, including TLS and AUTH"
+HOMEPAGE="http://www.jetmore.org/john/code/swaks"
```
Comment (juippis) [inline net-mail/swaks/swaks-20201010.0.ebuild:9]: Please also add the Github page here as 2ndary HOMEPAGE.

File: net-mail/swaks/swaks-20201010.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+SRC_URI="http://www.jetmore.org/john/code/swaks/${P}.tar.gz"
+LICENSE="GPL-2"
```
Comment (juippis) [inline net-mail/swaks/swaks-20201010.0.ebuild:11]: They seem to be using `GPL-2+` precisely.

File: net-mail/swaks/swaks-20201010.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+	dev-perl/CGI
+	dev-perl/DBI
+	dev-perl/Email-Send
+	dev-perl/Email-Valid
+	dev-perl/Params-Validate
+	dev-perl/URI
+	ssl? ( dev-perl/IO-Socket-SSL )
+	virtual/perl-Data-Dumper
+	virtual/perl-Digest-SHA
+	virtual/perl-Getopt-Long
```
Comment (juippis) [inline net-mail/swaks/swaks-20201010.0.ebuild:34]: Please sort this list alphabetically, and remove the forbidden `ssl?` line moving it to `optfeature`.

**Comment by alarig** [comment]
Hi,

I pushed fixes for your comments, and renamed the file to `swaks-20201014.0.ebuild` as this new version is a bugfix for `20201010.0`.

Please let me know if you have other remarks.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**